### DISCUSSION TITLE
Read LCD with "clamp" around numbers

### DISCUSSION THREAD

**Discussion by taxfromdk** [jomjol/AI-on-the-edge-device#2867] [Q&A]
I have a kamstrup meter where I read the following image:

https://i.imgur.com/w2TrHWA.png

The meter has a clamp surrounding the three last digits, and that triggers digits like 1 to be read as 7. 

My ROI's are defined like this:
https://i.imgur.com/3f1NrHa.png

Any advice on how to proceed?

**Comment by caco3** [comment]
You could put white tape around the numbers, hiding the clamps.
Or try to adjust the ROIS so the clamps are outside of the used area.

**Comment by monsivar** [comment]
I got a kamstrup FlowIQ 2200 with the "clamps" and no problem reading the numbers using the dig-class11_1700_s2 model

<img width="287" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/32893134/3f553bc1-0251-4587-bc37-5554c65b622b">

**Reply by Commifreak** [reply]
> I got a kamstrup FlowIQ 2200 with the "clamps" and no problem reading the numbers using the dig-class11_1700_s2 model

Same at my side:

![grafik](https://github.com/user-attachments/assets/3bb936ac-2c01-4f89-86b5-20b004b374ed)
![grafik](https://github.com/user-attachments/assets/0ee51e57-1f42-483a-bd22-0952ed1d6fc5)

Using mentioned model. Just set up, so no long term experience, but it looks good so far.

**Comment by taxfromdk** [comment]
Thanx, I will give it a try

Den man. 19. feb. 2024 kl. 20.39 skrev monsivar ***@***.***>:

> I got a kamstrup FlowIQ 2200 with the "clamps" and no problem reading the
> numbers using the dig-class11_1700_s2 model
> image.png (view on web)
> <https://github.com/jomjol/AI-on-the-edge-device/assets/32893134/3f553bc1-0251-4587-bc37-5554c65b622b>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jomjol/AI-on-the-edge-device/discussions/2867#discussioncomment-8521496>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFFVEUBGVMKFUTIUWLE66LYUOTABAVCNFSM6AAAAABCZEKWHKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMRRGQ4TM>
> .
> You are receiving this because you authored the thread.Message ID:
> <jomjol/AI-on-the-edge-device/repo-discussions/2867/comments/8521496@
> github.com>
>


-- 
Jesper Taxbøl
+45 61627501
